kratos/python_scripts/analysis_stage.py

```diff
@@ -78,6 +78,7 @@ def Initialize(self):
             process.ExecuteInitialize()
 
         self._GetSolver().Initialize()
+        self.Check()
```

Comment:
I appreciate that there may be issues with this...however shouldn't we do it BEFORE the initialize?

to make it clear, I DO SEE that you are not changing the current behaviour :-)

Reply:
Hm I honestly don't know what would be better ... As you said I just unified the current behavior

Either way, if we decide to change this then we will have to do it only in one place if this is merged :)
Therefore I suggest that I merge this and we can discuss when to call `Check` separately
**I will merge on Friday unless someone objects**

If you want I can also open an issue regarding when to call `Check`

Thanks btw for discussing this in the committee